src/backend/dx11/src/device.rs

```diff
@@ -677,15 +681,38 @@ impl Device {
 
         match info.view_kind {
             image::ViewKind::D2 => {
-                desc.ViewDimension = d3d11::D3D11_DSV_DIMENSION_TEXTURE2D;
-                *unsafe { desc.u.Texture2D_mut() } = d3d11::D3D11_TEX2D_DSV { MipSlice }
+                match info.kind {
+                    image::Kind::D2(_, _, _, 1) => {
```

Comment:
seems unnecessary to repeat this match, it could be simpler to just do something like `if info.kind.num_samples() > 1 {..} else {...}` in both places